rules.bzl

```diff
@@ -25,8 +25,8 @@ def _android_ndk_repository_impl(ctx):
     """
     ndk_path = ctx.attr.path or ctx.os.environ.get("ANDROID_NDK_HOME", None)
     if not ndk_path:
-        fail("Either the ANDROID_NDK_HOME environment variable or the " +
-             "path attribute of android_ndk_repository must be set.")
+        ctx.file("BUILD.bazel", "# No path to Android NDK was provided", executable = False)
```

Comment:
I think we'll want to have something similar to https://github.com/bazelbuild/rules_android/blob/12c9f74819fefc3a5544f953ee6aab913cf5627a/rules/android_sdk_repository/empty.template.bzl#L57

so that there's a bigger hint about what's going on when there isn't an ndk set up. And this would probably be better in a separate change

Reply:
Fair enough, I'll open another PR and remove it from this one.